Add initial security policy (#342)
Aorimn/dislocker: SECURITY.md

# Security Reporting

If you wish to report a security vulnerability privately, we appreciate your diligence. Please follow the guidelines below to submit your report.

## Reporting

To report a security vulnerability, please provide the following information:

1. **PROJECT**
   - Include the URL of the project repository - Example: <https://github.com/Aorimn/dislocker>

2. **PUBLIC**
   - Indicate whether this vulnerability has already been publicly discussed or disclosed.
   - If so, provide relevant links.

3. **DESCRIPTION**
   - Provide a detailed description of the security vulnerability.
   - Include as much information as possible to help us understand and address the issue.

> Send this information, along with any additional relevant details, to <email AT somewhere or other channel>.

Reviewer: Thanks for that whoe file.

Here's where they can send this information:
```suggestion
Send this information, along with any additional relevant details, to <aorimn AT gmail DOT com>.
```
